src/main/java/ba/edu/ibu/job/search/platform/rest/controllers/CompanyController.java

```diff
@@ -1,10 +1,15 @@
 package ba.edu.ibu.job.search.platform.rest.controllers;
 import ba.edu.ibu.job.search.platform.core.model.Company;
 import ba.edu.ibu.job.search.platform.core.service.CompanyService;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import ba.edu.ibu.job.search.platform.rest.dto.CompanyDTO;
+import ba.edu.ibu.job.search.platform.rest.dto.CompanyRequestDTO;
+import ba.edu.ibu.job.search.platform.rest.dto.UserDTO;
+import ba.edu.ibu.job.search.platform.rest.dto.UserRequestDTO;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
 import java.util.List;
 
 @RestController
```

Comment:
You should add an annotation to make this work in Swagger.

@SecurityRequirement(name = "JWT Security")

Reply:
Added.